src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -344,13 +408,13 @@ dhallPBName name = Qual (Module hsDhallPB) (HsIdent name)
 
 dhallInterpretInstDecl :: String -> HsDecl
 dhallInterpretInstDecl typeName =
-  instDecl_ (dhallPBName "FromDhall")
+  instDecl_ (dhallPBName "Interpret")
```

Comment:
The tests are failing because of this change.  This may be due to incorrectly resolving a merge conflict when merging `master` into your branch.  If you change this back to `FromDhall`/`ToDhall` then the test should no longer fail on that

Reply:
@Gabriel439 Yes I noticed this and though something is weird. I run `git add --patch` to review my changes before I stage them and this one made me wonder what I did. Thank you!